docs/software/integrations/mqtt/index.mdx

```diff
@@ -6,21 +6,39 @@ sidebar_position: 6
 description: Bridging mesh networks over the internet and integrating Meshtastic protocols with popular technologies such as Home Assistant, Node Red, and Adafruit IO.
 ---
 
-## Bridging networks
+## Public MQTT Server
 
 ![Common MQTT Layout](/img/software/mqtt/mqtt.webp)
 
-Meshtastic networks in different locations beyond the reach of LoRa can be easily bridged together using MQTT. The simplest option is to connect your mesh to the official Meshtastic MQTT broker. This makes your devices appear on the world map, and provides a copy of your mesh traffic, translated into JSON. All you have to do to join the public MQTT server is to Enable MQTT and set Uplink and Downlink on the channels that you want to share over MQTT. The default device configuration using the public MQTT Server is encrypted.
+The Meshtastic project provides a public MQTT service that users can connect to, with certain restrictions in place to ensure network stability. This service allows Meshtastic devices to bridge over the internet, providing global connectivity for remote networks.
 
-You can also specify your own private MQTT broker to bridge mesh networks together, via the internet (or just a local IP network).
+For instructions on connecting to the public MQTT server, please refer to [Connect to the Default Public Server](/docs/configuration/module/mqtt/#connect-to-the-default-public-server).
 
-You can find the settings available for MQTT [here](/docs/configuration/module/mqtt).
+### Restrictions on the Public MQTT Server
 
-:::important
-When MQTT is turned on, you are potentially broadcasting your entire mesh's traffic onto the public internet. This includes messages and position information.
+To maintain optimal performance and protect LoRa meshes, traffic restrictions are currently applied to the public MQTT server.
 
-The default channel (LongFast) on the public MQTT server usually has a lot of traffic. Your device may get overloaded and may no longer function properly anymore. It is recommended to use a different channel or to use your own MQTT server if you experience issues.
-:::
+#### Zero-Hop Policy
+
+Traffic from the public MQTT server does not fully propagate through local mesh networks. Directly connected nodes will receive the data, but due to the zero-hop policy, it will not spread further to other nodes within the local mesh network.
```

Comment:
Does zero-hop apply to all traffic, or only the default psk?

Reply:
Ah news to me.  You may wish to remove or amend [this announcement](https://discord.com/channels/867578229534359593/871558934916390953/1260642727335428176).